recipes/treedecomp/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "treedecomp" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5ab35269a6e1335b8ea7fc319448319e258e6ea00dbc4c5b65436623864f0fce
+
+build:
+  number: 0
```

Comment:
I think this can be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python